test/azlinux_test.go

```diff
@@ -394,6 +394,7 @@ echo "$BAR" > bar.txt
 				Post: &dalec.PostInstall{
 					Symlinks: map[string]dalec.SymlinkTarget{
 						"/usr/bin/src1": {Path: "/src1"},
+						"/usr/bin/src2": {Path: "/non/exists/dir/src2"},
```

Comment:
I think this is not working because you haven't added `src2` (and also src2 is a dir) to the list of binary artifacts.
I think you'll need to add another source file with the content you want to check for.

Reply:
I guess I looked at the wrong line when I looked up `src2`: https://github.com/Azure/dalec/blob/main/test/azlinux_test.go#L1026

Thanks for spotting that